lib/site-inspector.rb

```diff
@@ -55,6 +55,10 @@ def typhoeus_defaults
           "User-Agent" => "Mozilla/5.0 (compatible; SiteInspector/#{SiteInspector::VERSION}; +https://github.com/benbalter/site-inspector)"
         }
       }
+      unless @typhoeus_options.nil?
```

Comment:
If you're okay with truthiness, I recommend `if @typhoeus_options` over `unless @typhoeus_options.nil?`.